### DISCUSSION TITLE
Options for options picker from different table

### DISCUSSION THREAD

**Discussion by mslourens** [Budibase/budibase#2240] [General]
Say I have a database with a table Products and a table Categories. The products table contains a relationship column with the categories table (one product to many categories). Is it possible to add an options picker to the products form using the rows in the categories table as options and store them in the relationship column of the products table?

**Comment by mjashanks** [comment]
Like this?

![image](https://user-images.githubusercontent.com/3524181/128077926-b0c48ee7-0389-457a-b6e0-0458abd9973a.png)

I setup as you described, and updated all the form fields on the `Field Group`. It added this by default.

![image](https://user-images.githubusercontent.com/3524181/128078094-a362564b-0056-4e5a-9668-643565e78542.png)

**Reply by mslourens** [reply]
Clicking the Update form fields button did create a select with the correct fields as options, but when I tried to add an Options Picker, it did not list my column. What I needed was a Relationship picker instead of an Options Picker.
